spec/lib/special_route_publisher_spec.rb

```diff
@@ -2,33 +2,6 @@
 require_relative "../../lib/special_route_publisher"
 
 RSpec.describe SpecialRoutePublisher do
-  describe "publish" do
```

Comment:
I think it might be better to remove the test for the rake task, and perhaps add a test for `find_route` in here instead, so that tests are together?

Reply:
yep. uno momento!